tools/bin/hawqregister

```diff
@@ -855,6 +888,79 @@ class HawqRegister(object):
         logger.info('Hawq Register Succeed.')
 
 
+class HawqRegisterPartition(HawqRegister):
```

Comment:
Also, where is the register function of this class? Can we make the interface consistent? For example, 
```
def __init__(self, options, table, utility_conn, conn, failure_handler):
    HawqRegister.__init__(self, options, table, utility_conn, conn, failure_handler)

def register(self):
    HawqRegister.register(self)
```